Source/Shaders/PostProcessFilters/FXAA3_11.glsl

```diff
@@ -0,0 +1,2074 @@
+/**
```

Comment:
Just make this the file in the `ThirdParty` directory.  Git history will have the original and this way we don't have any third-party code outside of the `ThirdParty` directory.